Sources/SwiftParser/Declarations.swift

```diff
@@ -30,13 +46,15 @@ extension TokenConsumer {
       _ = subparser.consumeAttributeList()
     }
 
-    if subparser.currentToken.isKeyword {
+    if subparser.currentToken.isKeyword ||
+        subparser.currentToken.tokenKind == .identifier {
```

Comment:
Do we even need this `if` condition? AFAICT it should be handled by the `while` condition already.